src/development/LocalTest/appsettings.Docker.json

```
@@ -0,0 +1,7 @@
+{
+  "LocalPlatformSettings": {
+    "LocalAppMode": "http",
```

Oops. need to use host.docker.internal if the app is running on the host system.
```suggestion
    "LocalAppMode": "http",
    "LocalAppUrl": "http://host.docker.internal:5005",
```